src/administration/from_scratch.md

```diff
@@ -322,46 +314,21 @@ Let's Encrypt certificates should be renewed automatically, so add the line belo
 @daily certbot certonly --nginx --cert-name example.com -d example.com --deploy-hook 'nginx -s reload'
 ```
 
-Finally, add the nginx config file. After downloading, you need to replace some variables in the file.
+Finally, add the Nginx virtual host config file. After downloading, you need to replace some variables in the file.
 
-```bash
-curl https://raw.githubusercontent.com/LemmyNet/lemmy-ansible/main/templates/nginx.conf \
+```
+sudo curl https://raw.githubusercontent.com/LemmyNet/lemmy-ansible/main/templates/nginx.conf \
     --output /etc/nginx/sites-enabled/lemmy.conf
 # put your actual domain instead of example.com
-sed -i -e 's/{{domain}}/example.com/g' /etc/nginx/sites-enabled/lemmy.conf
-sed -i -e 's/{{lemmy_port}}/8536/g' /etc/nginx/sites-enabled/lemmy.conf
-sed -i -e 's/{{lemmy_ui_port}}/1234/g' /etc/nginx/sites-enabled/lemmy.conf
-nginx -s reload
+sudo sed -i -e 's/{{domain}}/example.com/g' /etc/nginx/sites-enabled/lemmy.conf
+sudo sed -i -e 's/{{lemmy_port}}/8536/g' /etc/nginx/sites-enabled/lemmy.conf
+sudo sed -i -e 's/{{lemmy_ui_port}}/1234/g' /etc/nginx/sites-enabled/lemmy.conf
+sudo systemctl reload nginx
 ```
 
-If you don't need HTTPS on nginx, the below file can be used as an updated template. Otherwise, feel free to merge changes from it to your own version.
+The Nginx template in the [lemmy-ansible repo](https://raw.githubusercontent.com/LemmyNet/lemmy-ansible/main/templates/nginx.conf) might need some more tweaking for this setup. It is necessary to help Nginx differentiate between the calls to the Lemmy API and the UI. Use the following snippet in the `server` block that deals with HTTP connections to Nginx. Replace `8536` with the Lemmy backend port, '1234' with the UI one.
```

Comment:
Ah the file was moved/rewritten so its not working anymore. [This version](https://github.com/LemmyNet/lemmy-ansible/blob/0a02a429d66120da8a054a29ed3fc4d5d3a708a6/templates/nginx.conf) is the one that was originally linked here. I guess it makes sense to copy directly into this file so that it stays consistent with the rest of the guide.

Reply:
Gotcha. Both comments have been taken care of.